core/src/main/java/com/klaytn/caver/tx/Account.java

```diff
@@ -49,4 +49,31 @@ public static Account create(Caver caver, KlayCredentials klayCredentials, int c
 
         return Account.create(caver, transactionManager);
     }
+
+    /**
+     * @deprecated  <p>In caver-java 1.0.0, we provided static methods to send transactions for `ValueTransfer`, `Account`, `Cancel`, and `SmartContract` classes. the static methods were removed for extensibility.</p>
```

Comment:
```suggestion
     * @deprecated  <p>In caver-java 1.0.0, we provided static methods to send transactions for `ValueTransfer`, `Account`, `Cancel`, and `SmartContract` classes. the static methods will be removed.</p>
```

Reply:
Please check other tx types.

Reply:
fixed.  and change '{will be removed in next version}' to 'This deprecated method can be used only for Baobab Testnet'